conf/app.routes

```diff
@@ -4,6 +4,7 @@ POST       /movements/departures
 GET        /movements/departures                                                controllers.DeparturesController.getDepartures()
 
 GET        /movements/departures/:departureId                                   controllers.DeparturesController.get(departureId: DepartureId)
+GET        /movements/departures/:departureId/mrn-allocated-message             controllers.DeparturesController.getMrnAllocatedMessage(departureId: DepartureId)
```

Comment:
Not very restful, if we do it like this, we'll end up with an endpoint for every message type!  This should be a filter on the /messages endpoint.  e.g. .../messages?type=IE028.

Reply:
Sorry forgot to close this, its not needed